java/ql/src/utils/stub-generator/Stubs.qll

```diff
@@ -391,6 +402,24 @@ private string stubMember(Member m) {
   )
 }
 
+private string stubAnnotation(Annotation a) {
+  if exists(a.getAValue())
+  then result = a.toString() + "(" + concat(stubAnnotationValue(a.getAValue()), ",") + ")"
+  else result = a.toString()
+}
+
+private string stubAnnotiationSimpleValue(Expr value) {
+  result = value.(FieldAccess).getField().getQualifiedName() or
+  result = value.(Literal).toString()
+}
```

Comment:
Would it be necessary to cover the following as well?
- annotations containing annotations, e.g. `@Outer(@Inner)`
- `TypeLiteral` (which does not extend CodeQL `Literal`), e.g. `String.class`
- `CompileTimeConstantExpr`, which is neither a `Literal` nor a `FieldRead`, e.g. negative literals or arithmetic operations